helm/thingsboard/templates/node.yaml

```diff
@@ -99,6 +99,11 @@ spec:
               name: {{ .Release.Name }}-node-config
             - mountPath: /var/log/thingsboard
               name: {{ .Release.Name }}-node-logs
+{{- if .Values.node.ruleChain.configmapName }}
```

Comment:
does the if statement work when configmapName is a string and not a boolean?

Reply:
It will work according to the docs: https://helm.sh/docs/chart_template_guide/control_structures/#:~:text=A%20pipeline%20is%20evaluated%20as%20false%20if%20the%20value%20is%3A

Basically empty or null value would be considered false, if there is a string true.

Reply:
ok, perfect then